openpype/hosts/blender/utility_scripts/set_workfile_publish_time.py

```diff
@@ -0,0 +1,13 @@
+import bpy
+
+from openpype.lib.log import Logger
+from openpype.lib.dateutils import get_timestamp
+
+if __name__ == "__main__":
+    log = Logger().get_logger()
+    log.debug("Setting workfile last publish time")
```

Comment:
For a debug log, I think printing the value of the last publish time would be nice.